### DISCUSSION TITLE
Does Phosphor have an SVG sprite?

### DISCUSSION THREAD

**Discussion by iamdarkle** [phosphor-icons/homepage#648] [General]
I was wondering if Phosphor includes a `.symbol.svg` file somewhere to facilitate SVG sprite usage (via `<use>`), like, for example:

```
<svg class="">
    <use xlink:href="your-path/icon.symbol.svg#admin-fill"></use>
</svg>
```

**Comment by rektdeckard** [comment]
Not yet. We can look into it for the next release, or we'd be happy to review a contribution!
